articles/understanding-the-basics-of-responsive-web-design/index.md

```diff
@@ -0,0 +1,118 @@
+
+**Step 5:** Install WordPress
+When you select a hosting plan, the hosting company will send a link to the email you used for registration. This link will take you to the hosting dashboard. When you fill in your credentials, you will find a section called ‘WordPress install’. Click on the WordPress icon and go through the instructions. 
+
+Another screen will pop up requiring you to fill in details such as your site’s description and desired credentials. After filling in all this information, click on ‘install’. Installing WordPress means that you can now access the WordPress dashboard using the registered credentials. 
+
+**Step 6:** Installing a theme
+In this step, you need to install the WordPress theme that you chose in step 4. In the WordPress dashboard, follow this selection sequence: **appearance**-> **themes**-> **add new**. Click on 'upload theme'. Click on 'install' to begin the installation. When this is done, click on 'activate'. 
+
+**Step 7:** Install plugins
+When your chosen theme has been activated, you will find a link written ‘begin installing plugins’. Click on this link to begin the installation process. After selecting all the plugins, click on ‘install’. Any inactive plugins can be activated by following this sequence: **plugins**-> **all plugins**-> **inactive**. Click on ‘activate’. 
+
+**Step 8:** Import demo content
+You can import the demo content to get an overview of how your site will look like. Go to 'import' and choose a demo design. After selecting the contents needed, click on ‘install’. 
+
+**Step 9:** Customize your site
+This can be done by replacing the demo content with the content you prepared in step 3. Simply go to the part that you want to change and enter your intended text. Change the settings accordingly to suit your needs. 
+
+### Examples of Responsive Websites
+Here are three examples of popular responsive websites.
+
+#### 1. BMW
+BMW is a multinational automobile company that produces motorcycles and luxury vehicles. The website for this company consists of many background videos as well as images. The following is a screenshot of this website on a laptop. 
+
+![BMW via Laptop Browser](/engineering-education/understanding-the-basics-of-responsive-web-design/bmw-via-laptop-browser.png)
+
+All this content responds well to various screen sizes. The website appears as follows on a mobile device.
+
+![BMW via Mobile Browser](/engineering-education/understanding-the-basics-of-responsive-web-design/bmw-via-mobile-browser.jpg)
+
+You can access all the icons even with a mobile device. When using the laptop browser, the main icons appear at the top of the website. When using the mobile device, you can access these icons using a drop-down menu at the top left corner of the website. 

```

Comment:
It is worth mentioning that it is called a Hamburger menu

Reply:
Done.